src/main/msp/msp.c

```diff
@@ -3254,13 +3254,13 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         }
         if (sbufBytesRemaining(src) >= 6) {
             // Added in MSP API 1.40
-            rxConfigMutable()->rcInterpolationChannels = sbufReadU8(src);
+            sbufReadU8(src); //was rxConfigMutable()->rcSmoothingChannels = sbufReadU8(src);
```

Comment:
Needs space after `//`. ` = sbufReadU8(src);` in the comment is not needed. 
This parameter was also removed in API version 1.44